src/app/withdraw/[txid]/components/withdrawal-status.tsx

```diff
@@ -63,6 +72,20 @@ function Content(initialData: Props) {
       return 5000;
     },
   });
+
+  const router = useRouter();
+
+  useEffect(() => {
+    // if the route is request id format do nothing
+    if (txid.length < 64) {
+      return;
+    }
+
+    // if it is a tx id and we have a request id navigate to request id
+    if (requestId) {
+      router.push(`/withdraw/${requestId}`);
```

Comment:
mmm there's a nuisance with this: the request id does depend on the init withdrawal submission order, and on reorgs you cannot assume it's going to play in the same order (even with tx replay), so I think we shouldn't redirect to request id until the request id "is final"; on the signers we assume 6 bitcoin blocks to be final enough (it's not related to the signers, just suggesting a number).

We can address this separately from here, just flagging it

Reply:
so the withdrawal id is never 100% deterministic until enough finality is reached on bitcoin 6 blocks is the minimum